code/modules/shuttle/computer.dm

```diff
@@ -75,6 +74,69 @@
 	if(port && (shuttleId == initial(shuttleId) || override))
 		shuttleId = port.id
 
+/obj/structure/machinery/computer/shuttle/ert
+	name = "transport shuttle"
+	desc = "A transport shuttle flight computer."
+	icon_state = "syndishuttle"
+	req_access = list()
+	breakable = FALSE
+
+/obj/structure/machinery/computer/shuttle/ert/tgui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if (!ui)
+		ui = new(user, src, "NavigationShuttle", name)
+		ui.open()
+
+/obj/structure/machinery/computer/shuttle/ert/ui_state(mob/user)
+	return GLOB.not_incapacitated_and_adjacent_strict_state
+
+/obj/structure/machinery/computer/shuttle/ert/ui_data(mob/user)
+	var/obj/docking_port/mobile/emergency_response/ert = SSshuttle.getShuttle(shuttleId)
+	var/list/docks = SSshuttle.stationary
+	. = list()
+	.["destinations"]=list()
+	for(var/obj/docking_port/stationary/emergency_response/dock in docks)
+		var/can_dock = ert.canDock(dock)
```

Comment:
![image](https://user-images.githubusercontent.com/66756236/208640482-b05ae531-df67-41c3-be4f-d27a0e6fd342.png)
this check does not account for shuttles already being in transit to the dock, so you can send one shuttle, then send another to the same dock, making the second shuttle stuck in hyperspace forever

Reply:
I've been able to resolve this - a dock is considered unavailable if someone already has it as a destination